src/codegen/function_body_chunk.rs

```diff
@@ -1308,14 +1308,20 @@ fn add_chunk_for_type(
     let type_ = env.type_(typ_);
     match *type_ {
         library::Type::Fundamental(ref x) if !x.requires_conversion() => true,
-        library::Type::Fundamental(library::Fundamental::Boolean)
-        | library::Type::Fundamental(library::Fundamental::UniChar) => {
+        library::Type::Fundamental(library::Fundamental::Boolean) => {
             body.push(Chunk::Custom(format!(
                 "let {0} = from_glib({0});",
                 par.name
             )));
             true
         }
+        library::Type::Fundamental(library::Fundamental::UniChar) => {
+            body.push(Chunk::Custom(format!(
+                "let {0} = std::convert::TryFrom::try_from({0}).unwrap();",
```

Comment:
Maybe using an `expect` in here instead? :)

Reply:
We can, IIUIC, the `unwrap` will print the `CharTryFromError` message in this case.

Reply:
I want proof. 😄

Reply:
:) try it yourself: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=bb2e9c3697d3567db45085c521c3d8c3

:D

Reply:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: CharTryFromError(())', src/main.rs:4:54
```

Sorry but this is just not good enough. For a "novice" rust user, this isn't helpful at all. Please use `expect`.

Reply:
Yeah unclear why `unwrap()` prints the `Debug` impl of the error instead of the `Display` impl. That's quite useless behaviour.